mpas_analysis/analysis_task_template.py

```diff
@@ -0,0 +1,133 @@
+'''
+This is an example analysis task to be used as a template for new tasks.
+It should be copied into one of the component folders (`ocean`, `sea_ice`,
+`land_ice`, etc.) and modified as needed.
```

Comment:
Add something along these lines:
`Specific parts or variable names that need to be modified by the developer are explicitly indicated. All the others are to be left unchanged.`